data/campaigns/Winds_of_Fate/scenarios/07_Harvest.cfg

```diff
@@ -487,7 +486,7 @@ My diary is very personal, Vonel!"
             speaker=narrator
             message= _ "31st day of Leaffall
 
-...those saurians may smell of the swamp they crawl from, but they shall make a worthy sacrifice to Zhangor! I have extended a formal invitation to Arinexis, a saurian ambassador, or what passes for one amongst their primitive and malodorous kind. I shall show this saurian and its entourage every courtesy and tempt them with all the delights of elven cuisine— laced with a sleeping potion of my own design..."
+...those saurians may smell of the swamp they crawl from, but they shall make a worthy sacrifice to Vanar! I have extended a formal invitation to Arinexis, a saurian ambassador, or what passes for one amongst their primitive and malodorous kind. I shall show this saurian and its entourage every courtesy and tempt them with all the delights of elven cuisine..."
```

Comment:
I don't actually care one way or another, but I'm curious why you chose to remove the sleeping potion reference.

Reply:
It's a bit heavy handed.

Reply:
Without the sleeping potion, it is unclear how she plans to capture the saurians and drakes though, no? Since she never required them to disarm and some of the saurians and drakes still fight quite well without using weaponry at all.

(This was also why she tried a couple times to get her 'guests' to eat somethings.)

Reply:
In other words, the intention is that although she _did_ use a sleeping potion, she's not coming right out and saying it… right?

Reply:
You can probably infer that she did something to the food and drink though exactly what she did you can just guess. Maybe it was a sleeping potion or maybe something else, presumably not very good for her guests though.